samples/wifi/scan/src/main.c

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ * @brief WiFi scan sample
+ */
+
+#include <zephyr/logging/log.h>
+#include <nrfx_clock.h>
+#include <zephyr/zephyr.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/init.h>
+
+#include <zephyr/net/net_if.h>
+#include <zephyr/net/wifi_mgmt.h>
+#include <zephyr/net/net_event.h>
+
+#define WIFI_SHELL_MODULE "wifi"
+
+#define WIFI_SHELL_MGMT_EVENTS (NET_EVENT_WIFI_SCAN_RESULT |		\
+				NET_EVENT_WIFI_SCAN_DONE)
+
+LOG_MODULE_REGISTER(scan, CONFIG_LOG_DEFAULT_LEVEL);
```

Comment:
use the sample name in the logger.

Reply:
@krish2718  I've written the sample name "scan" in the logger. 
Is there anything that I can modify here?

Reply:
No.